dev/local-environment/configurations/partner-chains-setup/entrypoint.sh

```diff
@@ -0,0 +1,233 @@
+#!/bin/bash
+
+apt -qq update &> /dev/null
+apt -qq -y install jq ncat &> /dev/null
+
+cp /usr/local/bin/partner-chains-node /partner-chains-node
```

Comment:
Currently we have 3 ways of building docker images in this one repo:
* in Earthly
* in dev/docker
* in CD github workflow

I have tested this entrypoint with each of them and it was able to setup dependencies and execute partner-chains-node binary (GLIBC problems not present) commands.